src/main/java/com/hillel/items_exchange/security/jwt/JwtAuthenticationEntryPoint.java

```diff
@@ -15,15 +10,14 @@
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
 import java.util.Locale;
+import java.util.Objects;
 
 @Component
 public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint {
-    private ObjectMapper mapper;
     private MessageSource messageSource;
 
     @Autowired
-    public JwtAuthenticationEntryPoint(ObjectMapper mapper, MessageSource messageSource) {
```

Comment:
You should make correct merge of master branch here!

Reply:
It's not a mistake. This is an improvement in functionality. After these changes we can get an extended error message in Response body on the front-end side (like a status, number of error, time etc. in JSON format)